pizza-facts-&-trivia.md

```diff
@@ -45,3 +45,5 @@ Delight in intriguing and lesser-known facts that will surprise and entertain yo
   - 100 acres
 - What unusual pizza topping is popular in Japan?
   - Mayonnaise
+- What unusual pizza topping can be found in France, Belgium or Italy ?
```

Comment:
This is a great fact! There's an extra space before the ? : `Italy ?` Would you mind removing it and we'll get it merged in?

Reply:
Hi @BekahHW , you are right ! I will correct.
For your information, in France, we do add a space before any mark (!, ?, : ...).
Have a nice day !

Reply:
@Pondy007 that's interesting! Thanks for letting me know.